openedx_webhooks/github/dispatcher/actions/new_pull_request_helpers/ospr_jira.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+from .....lib.jira.decorators import inject_jira
+
+
+@inject_jira
+def create_ospr(jira, event, workflow_stage='Needs Triage'):
+    # TODO: implement
```

Comment:
Obviously this still needs to be implemented, it should return `{'label': 'OSPR-1234', 'url': 'https://xyz.com'}`.

The idea of `workflow_stage` is to be able to put the OSPR directly into a specific workflow stage. For example, if edX requires CLA, then it can go directly to the "Awaiting Author" stage, instead of someone having to triage.

Reply:
@nedbat The only remaining thing is to create the actual OSPR in JIRA:

1. This should probably be moved to the `openedx_webhooks.jira.tasks` module, since it has to do with JIRA, not GitHub.
2. You can see [`update_latest_github_activity`](https://github.com/edx/openedx-webhooks/blob/development/openedx_webhooks/jira/tasks.py#L48) for clues on how this function should be implemented.